src/q2g-ext-selector.ts

```diff
@@ -3,10 +3,10 @@ import "css!./q2g-ext-selectorExtension.css";
 import * as qvangular from "qvangular";
 import * as qlik from "qlik";
 import * as template from "text!./q2g-ext-selectorExtension.html";
-import * as langDE from "text!./translate/de-DE/propertypanel.js";
-import * as langEN from "text!./translate/en-US/propertypanel.js";
+import * as langDE from "./translate/de-DE/propertypanel";
```

Comment:
@r-hannuschka thats an issue, I very shure that live (tsc compile on save) mode with qlik is not working any more after this change

Reply:
this is not a problem, i tested this out and it is working with simple tsc or q2g-build process. in both cases extension is working.

The mainproblem here is, since the qext file will now generated it is missing and must created manually.